test-integration/test-server/pom.xml

```diff
@@ -11,7 +11,7 @@
     <packaging>war</packaging>
 
     <properties>
-        <version.infinispan.hotrod>12.0.0.Dev04</version.infinispan.hotrod>
+        <version.infinispan.hotrod>11.0.3.Final</version.infinispan.hotrod>
```

Comment:
Can we consolidate all possible version properties inside parent pom?

Reply:
`test-server` module doesn't have parent as it's supposed to be build as a standalone WebServer application with rest triggered endpoints. The application is built in the OpenShift and the parent directory isn't loaded there.

Reply:
Good, I see. It would be nice to keep versions in one place to avoid typos if possible.